.travis.yml

```diff
@@ -50,7 +49,6 @@ matrix:
 
     - os: linux
       rust: stable
-      sudo: required
```

Comment:
`sudo: required` for kcov. whereas for the root conf and for all other runners you should set `sudo: false`.